backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -94,8 +95,15 @@ def reshape_dataset_for_curation_api(dataset: dict, preview=False) -> dict:
         for asset in artifacts:
             if asset["filetype"] in (DatasetArtifactFileType.H5AD, DatasetArtifactFileType.RDS):
                 dataset["dataset_assets"].append(asset)
-    if dataset.get("processing_status"):
-        dataset["processing_status"] = dataset["processing_status"]["processing_status"]
+    if processing_status := dataset.pop("processing_status", None):
```

Comment:
nit comment: you know, using `.pop`, as you've done here, for all of these reassignments/reshapings actually makes more sense than using `.get` since it forces you to make sure you're actually reassigning (otherwise API response shape catches the error). I should've done this in some other places. 👍